cmd/new/new.go

```diff
@@ -56,13 +51,20 @@ func CreateCommand() *cobra.Command {
 		return utils.Find(formats, toComplete), cobra.ShellCompDirectiveNoSpace | cobra.ShellCompDirectiveNoFileComp
 	})
 
-	home, _ := homedir.Dir()
 	tmp.Flags().StringVar(&localTemplateCache, "templatepath", "", "location of installed templates")
 	viper.BindPFlag("templatepath", tmp.Flags().Lookup("templatepath")) //nolint:errcheck
-	viper.SetDefault("templatepath", filepath.Join(home, ".pdk", "pct"))
 	return tmp
 }
 
+func configureTemplatePath(cmd *cobra.Command, args []string) error {
+	log.Trace().Msg("PreRunE")
+	localTemplateCache = viper.GetString("templatepath")
+	execDir, _ := os.Executable()
```

Comment:
dont skip errors, at the very least log it

```suggestion
	execDir, err := os.Executable()
	
	if err != nil {
	  log.Error().Msgf("Unable to determine template path: %s", err.Error())
	}
```